build_tools/buildkite/pipelines/presubmit.yml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Note: this runs on "security: submitted" agents, since this pipeline itself is
+# submitted code and it fetches the scripts from the main branch.
+agents:
+  queue: "orchestration"
+  security: "submitted"
+
+# TODO: make it possible to test this pipeline on presubmit.
+
+steps:
+  - label: ":hiking_boot: Bootstrapping presubmit pipeline"
+    # If this PR is coming from the main repo we can use the configs from there
+    if: "!build.pull_request.repository.fork"
```

Comment:
For some reason this is triggering even if the repo isn't a fork. Will have to debug further

Reply:
I mean *is* a fork, but I think it's not now. Must have been an issue with updating the UI or something